Agents/FloodAssessmentAgent/setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='floodassessment',
-    version='0.0.1',
+    version='1.1.0',
```

Comment:
Could you please update to `pyderivationagent==1.4.4` and `py4jps==1.0.34`? This will incorporate the changes of the OntoDerivation namespace and some other bug fixes/improvements.

Reply:
Updating those pinned versions in `requirements.txt` should also work.

Reply:
Addressed by [dcaa7b0](https://github.com/cambridge-cares/TheWorldAvatar/pull/637/commits/dcaa7b0862cedec51f9953b823d43bfb28cb02bf)